pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -11,6 +11,8 @@ WARNING: DO NOT edit .pxi FILE directly, .pxi is generated from .pxi.in
 
 from pandas._libs.tslibs.util cimport get_c_string
 
+import pandas as pd
```

Comment:
Sorry should have clarified - can you cimport the sentinel? Should offer better performance